### PR TITLE
Register Subdomain: gavinferdinandchee.github.io

### PR DESCRIPTION
**Pull request by GavinFerdinandChee**
Added the domain: gavinferdinandchee.github.io

gavin.is-a.dev

### Requirements

- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. This requirement is optional if you are registering a domain for emails.
- [x] The website is reachable.
- [x] The CNAME record doesn't contain https:// or /.
- [x] There is sufficient information at the owner field.
    - You should have your email presented. If you don't want to share email, you can leave email an empty string ("") and add any other social such as Discord/Twitter/etc.

### Link to Website
https://gavinferdinandchee.github.io/

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please delete [this file](https://github.com/GavinFerdinandChee/register/blob/main/domains/gavinferdinandchee.github.io.json)

**Comment by GavinFerdinandChee** [comment]
> Please delete [this file](https://github.com/GavinFerdinandChee/register/blob/main/domains/gavinferdinandchee.github.io.json)

its done, thanks btw

**Review by wdhdev — APPROVED** [review]

File: domains/gavin.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Gavin Ferdinand Chee's Personal Portfolio",
+  "repo": "https://github.com/GavinFerdinandChee/GavinFerdinandChee.github.io",
+  "owner": {
+    "username": "GavinFerdinandChee",
+    "email_personal": "gavin.ferdinand@proton.me",
```
Comment (WilliamDavidHarrison) [inline domains/gavin.json:6]:
```suggestion
    "email": "gavin.ferdinand@proton.me",
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
Resolves failing checks

**Review by wdhdev — APPROVED** [review]

**Pull request merged**